content/en/docs/install-operate/help/missing-fields.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Help, Missing Fields in Falco Logs!
+description: Follow this guide to identify issues or misconfigurations that may be causing missing fields in Falco rules outputs
+weight: 30
+---
+
+## Action Items (TL;DR)
+
+- Read [Install and Operate](../../../install-operate/) Guides and review [falco.yaml](https://github.com/falcosecurity/falco/blob/master/falco.yaml) for necessary preconditions.
+- Refer to the relevant debugging guide based on suspected missing fields.
+- Acknowledge that certain missing fields or data in Falco are legitimate.
+
+## Background
+
+Many of the [Supported Output Fields](../../../reference/rules/supported-fields/) are derived from multiple events and mechanisms. To provide a more concrete explanation, for each spawned process, Falco extracts and derives fields from the `clone*/*fork/execve*` syscalls. Falco generates a struct in userspace, stores the relevant information within this struct, and then adds it to the process cache table in memory. If a process makes additional system calls during its lifetime, such as opening a file, in a Falco rule, you typically also export process fields — assuming we haven't missed the spawned process event and the information is available. These details extend to various use cases, and, in essence, dropped events can lead to missing fields as well as race conditions.
+
+As a result, Falco logs can never be perfect, and null values can occur. We are constantly aiming to improve the robustness in this regard. We encourage you to [contribute](/docs/contribute/) to the project if you encounter such cases or have improvement ideas. Also be aware that, unfortunately, missing fields can have different natures. Sometimes the field may be an empty string, or the string `<NA>`, or, if numeric, the default numeric value. These inconsistencies may be more difficult to address, as many Falco rules rely on legacy declarations.
+
+Furthermore, sometimes Linux may not operate exactly as expected. One concrete example is that shell built-ins like `echo` do not cause a new spawned process, and the `echo` command does not get logged with Falco. Similarly, if a base64 encoded string gets interpreted during decoding, you do not have the original base64 blob in the command args unless the command was passed with the `sh -c` flag. Lastly, some fields only work for certain kernel versions or system configs (e.g. [proc.is_exe_upper_layer](https://falco.org/docs/reference/rules/supported-fields/#field-class-process) requires a container overlayfs).
+
+## Missing Container Images
```

Comment:
@leogr I really hope we make this clearer, I'll post in one of the open issues that perhaps we should reconsider adding `host` to the image if it's a host process and for Falco 0.38 even `pod_sandbox_container` once the relevant libs PR is merged ...